README.md

```diff
@@ -21,7 +21,14 @@ See [picturingjordan.com](https://picturingjordan.com) for an example of this th
 - configurable keywords for every post
 
 ## Usage
-Clone the repository to your site's `themes` directory. Refer to [`exampleSite/config.toml`](https://github.com/alanorth/hugo-theme-bootstrap4-blog/blob/master/exampleSite/config.toml) for recommended configuration values.
+Clone the repository to your site's `themes` directory.
+
+### If you already using git
```

Comment:
Change section heading to [title case](http://individed.com/code/to-title-case/) and correct "your" to "you're".